client/src/pages/ResetPassword/ForgetPassword/ForgetPassword.tsx

```diff
@@ -0,0 +1,47 @@
+import { Grid, CssBaseline, Paper, Box, Typography } from '@material-ui/core';
+import { FormikHelpers } from 'formik';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import sendResetEmailAPI from '../../../helpers/APICalls/resetPassword';
+import EmailForm from './EmailForm/EmailForm';
+import { useHistory } from 'react-router-dom';
+import useStyles from './useStyles';
+
+export default function ForgetPassword(): JSX.Element {
+  const classes = useStyles();
+  const history = useHistory();
+  const { updateSnackBarMessage } = useSnackBar();
+  const handleSubmit = ({ email }: { email: string }, { setSubmitting }: FormikHelpers<{ email: string }>) => {
+    sendResetEmailAPI(email).then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else {
+        updateSnackBarMessage(data.success.message);
+        setTimeout(() => {
```

Comment:
As opposed to a timeout, maybe we can direct the user to the change password page when they click on the link in the email? :)

Reply:
yeah, this function is the current issue in this PR, if the user click the link, it will redirect to the login page. we can discuss to fix this problem.